pkg/controllers/user/rbac/handler_base.go

```diff
@@ -25,6 +26,7 @@ const (
 	projectIDAnnotation              = "field.cattle.io/projectId"
 	prtbByProjectIndex               = "authz.cluster.cattle.io/prtb-by-project"
 	prtbByProjecSubjectIndex         = "authz.cluster.cattle.io/prtb-by-project-subject"
+	prtbByUIDIndex                   = "authz.cluster.cattle.io/rtb-owner"
```

Comment:
is the value `rtb-owner` just a copy-paste mistake? I dont think it makes sense as a value

Reply:
hm. maybe it was intentional. i will circle back once ive read the rest of the cdoe